mystery_word.py

```diff
@@ -0,0 +1,219 @@
+import random
+from pyfiglet import Figlet
+import os
+import time
+
+
+f = Figlet(font='epic')
+
+
+def change_guess_counter(guess_counter, guessed_letter, game_word):
```

Comment:
You've got some repeated functions here. A good move would be to extract these into a common library both programs can use.